frontend/src/components/result/ResultDetailInfo.tsx

```diff
@@ -0,0 +1,52 @@
+import { css } from '@style/css';
+
+import { type SubmitResult } from './ResultList';
+
+export default function ResultInfo({ submitResult }: { submitResult: SubmitResult }) {
```

Comment:
단순하긴 하지만 
```suggestion
interface Props {
  submitResult: SubmitResult
}

export default function ResultInfo({ submitResult }: Props) {
```
와 같이 Props 인터페이스를 분리할 수 있을까요?